signature/signaturetest/algorithm.go

```diff
@@ -0,0 +1,18 @@
+package signaturetest
+
+import "github.com/notaryproject/notation-core-go/signature"
+
+// KeyTypes contains supported key type
+var KeyTypes = []signature.KeyType{signature.KeyTypeRSA, signature.KeyTypeEC}
+
+// GetKeySizes returns the supported key size for the named keyType
+func GetKeySizes(keyType signature.KeyType) []int {
```

Comment:
Where are we using this?

Reply:
It was used in signature/jws/envelope_test.go, also will be used in signature/cose/envelope_test.go in the following PR.